src/Components/Results/Results.tsx

```diff
@@ -265,18 +272,24 @@ const Results = ({ handleTextFieldChange }: ResultsProps) => {
         //we add that program's est value to its corresponding categoryValues key
         categoryValues[program.category.default_message] += program.estimated_value;
 
-        //if the program is a preschoolProgram, we also add it to the preschoolPrograms separately
-        if (program.category.default_message === preschoolProgramCategoryString) {
-          preschoolPrograms.numOfPreSchoolPrograms++;
-          preschoolPrograms.totalEstVal += program.estimated_value;
+        //if the program is in the preschoolProgramCategory, we also add it to the childCareTotalEstVal or youthAndEducationTotalEstVal separately
+        if (isPreschoolOrChildCareProgram) {
+          childCareTotalEstVal += program.estimated_value;
+        } else if (
+          program.category.default_message === childCareYouthAndEducationCategoryString &&
+          isPreschoolOrChildCareProgram === false
+        ) {
+          youthAndEducationTotalEstVal += program.estimated_value;
         }
       }
     }
 
-    if (preschoolPrograms.totalEstVal > 8640 && preschoolPrograms.numOfPreSchoolPrograms > 1) {
-      categoryValues[preschoolProgramCategoryString] = 8640;
+    if (childCareTotalEstVal > 8640) {
```

Comment:
Did you mean to remove the logic for if there is one preschool program to not cap the value?